sql/hive/src/test/scala/org/apache/spark/sql/hive/HiveSparkSubmitSuite.scala

```diff
@@ -453,6 +453,20 @@ object SetWarehouseLocationTest extends Logging {
       sparkSession.sql("drop table testLocation")
     }
 
+    // Also test data source table.
+    {
+      sparkSession.sql("create table testLocation (a int) using json")
+      val tableMetadata =
+        catalog.getTableMetadata(TableIdentifier("testLocation", Some("default")))
+      val expectedLocation = expectedWarehouseLocation + "/testlocation"
+      val actualLocation = tableMetadata.storage.locationUri.get
+      if (actualLocation != expectedLocation) {
```

Comment:
Don't you just want to `assert` this?

(Disregard my earlier comment, this doesn't relate to recent changes I made to the warehouse path)